components/patterns/form/_form.scss

```diff
@@ -44,6 +44,28 @@ $form-widths: (
   }
 }
 
+.form__footer-privacy {
+  @include respond-to(breakpoint(m)) {
+    display: none;
+  }
+
+  @include respond-to(breakpoint(l)) {
```

Comment:
I don't think this line is needed (and the one on line 64) as the code within the large breakpoint will already be covered by the code within the medium breakpoint.

Reply:
thanks. will test it.